core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -209,19 +391,15 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
           throttleTimeSensor = metrics.sensor(throttleTimeSensorName,
                                               null,
                                               ClientQuotaManagerConfig.InactiveSensorExpirationTimeSeconds)
-          throttleTimeSensor.add(metrics.metricName("throttle-time",
-                                                apiKey,
-                                                "Tracking average throttle-time per client",
-                                                "client-id",
-                                                clientId), new Avg())
+          throttleTimeSensor.add(throttleMetricName(quotaEntity), new Avg())
```

Comment:
Now that quota is a bit more complicated, would it be useful to log quotaEntity in debug level every time a sensor is created for debugging purpose?